numpy/core/src/umath/ufunc_object.c

```diff
@@ -2609,8 +2609,11 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
     for (i = 0; i < nin; ++i) {
         op_flags[i] = NPY_ITER_READONLY |
                       NPY_ITER_COPY |
-                      NPY_ITER_ALIGNED |
-                      NPY_ITER_OVERLAP_ASSUME_ELEMENTWISE;
+                      NPY_ITER_ALIGNED;
+        if (op[i] && PyArray_NDIM(op[i]) < 2) {
```

Comment:
What's special about 0d / 1d arrays?

Reply:
My fix was wrong. Any element-wise `gufunc` (one with an empty signature '(),()->()' will not hit this code path, it will not have `core_enabled`. Removing this flag.